libstuff/STCPNode.cpp

```diff
@@ -214,8 +245,18 @@ void STCPNode::postPoll(fd_map& fdm, uint64_t& nextActivity) {
                 _onDisconnect(peer);
                 if (peer->s->connectFailure)
                     peer->failedConnections++;
+                if(peer->s->ssl) {
+                    SDEBUG("MB SSL Closed");
+                    SSSLClose(peer->s->ssl);
+                    // Cleanup
+                    int ret;
+                    do ret = mbedtls_ssl_close_notify( &peer->s->ssl->ssl );
```

Comment:
I think we'll bug you about style on this one (just wrap the `do` block in curly braces).